tests/test_handlers.py

```diff
@@ -53,3 +56,34 @@ def test_all_handlers(input_dir: Path, output_dir: Path, tmp_path: Path):
     except subprocess.CalledProcessError as exc:
         runnable_diff_command = " ".join(diff_command)
         pytest.fail(f"\nDiff command: {runnable_diff_command}\n{exc.stdout}\n")
+
+
+@pytest.mark.parametrize(
+    "handler",
+    (
+        pytest.param(handler, id=handler.NAME)
+        for handler_map in _ALL_MODULES_BY_PRIORITY
+        for handler in handler_map.values()
+    ),
+)
+def test_missing_handlers_integrations_tests(handler):
+    handler_name = handler.NAME
+    handler_path = (
+        Path(inspect.getfile(handler.__class__))
```

Comment:
I would do it multiple steps, because a little bit hard to glance.
```suggestion
        handler_module_path = Path(inspect.getfile(handler.__class__))
        handler_test_path = handler_module_path.relative_to(HANDLERS_PACKAGE_PATH ).with_suffix("")
```